packages/authorization-token/src/trpc/authorization-tokens-router.ts

```diff
@@ -0,0 +1,71 @@
+import { initTRPC } from "@trpc/server";
+import type { AppContext } from "@webstudio-is/trpc-interface/server";
+import { db } from "../db";
+import { z } from "zod";
+
+const { router, procedure } = initTRPC.context<AppContext>().create();
+
+export const authorizationTokenRouter = router({
+  findMany: procedure
+    .input(
+      z.object({
+        projectId: z.string(),
+      })
+    )
+    .query(async ({ input, ctx }) => {
+      return await db.findMany({ projectId: input.projectId }, ctx);
+    }),
+  create: procedure
+    .input(
+      z.object({
+        projectId: z.string(),
+        relation: z.enum(["editors", "viewers", "builders"]),
```

Comment:
is it a convention to call those plural noun? 
what about: editor, viewer, builder
or verb: edit, view, build

Reply:
There are 2 type of things which are different.
- Relations i.e `User is "viewer" of Project` or `User is "editor" of Project`
- Permits i.e `User can "view" Project` _Permits are based on relation like User can "view" project if he has Relation "editor"_

From Zanzibar paper we see that Relations often has "editor", "viewer", "parent", "owner" keywords. 
But in [Ory Permission Language](https://www.ory.sh/docs/keto/reference/ory-permission-language) we usually describe that a little bit inverted, instead of `User is "viewer" of Project` there we declare 

```ts
class Project implements Namespace {
  related: {
    viewers: (User | Token)[]
    editors: (User | Token)[]
  }
```

i.e. Project has "viewers" with Users or Tokens

So for Relations we can use any form "viewer" or "viewers" for me 2nd preferred as gives a little bit more information -
"owner", "parent" - means we assume that single owner, single parent exists.
"viewers" - that multiple viewers can exist.

The downside of Zanzibar is that internally there are no unique constraints and in reality no way except name to show that I want single parent, single owner for the Project but many viewers, editors, and builders.

The downside of having `s` is that what if we will want many `parents`, many `owners` etc but the keyword is already an `owner`. I have no strong opinion what's better

Reply:
We should expect having multiple owners I think